### PR TITLE
#161779865 Refactors the facebook passport test

### PR DESCRIPTION
**Pull request by theghostyced**
#### What does this PR do?
Covers more test for the facebook passport file
#### Description of Task to be completed?
Writes more test to cover for the facebook passport file
#### How should this be manually tested?
git clone this branch
Add the `FACEBOOK_APP_ID` and `FACEBOOK_APP_SECRET` to your env
Run `npm run test`
#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[#161779865 ](https://www.pivotaltracker.com/story/show/161779865)
#### Screenshots (if appropriate)
Nil
#### Questions:
Nil

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -6,3 +6,7 @@ DB_TEST=
 DB_DEV=
 DB_PRODUCTION=
 DB_HOST=
+
+
+FACEBOOK_APP_ID=
+FACEBOOK_APP_SECRET=
```
Comment (Danielshow) [inline .env.sample]: @theghostyced, Newline needed here

Comment (theghostyced) [reply .env.sample]: Okay thanks @Danielshow. Done

Comment (V-84) [reply .env.sample]: Nice work daniel. Git says you need to add a newline

**Review by Danielshow — APPROVED** [review]
Looks good to me

**Pull request merged**